Description: Fix Makefile issue
 Makefile has commands that require browserify. Ignoring for now.
Author: Sarath M S <mail@sarathms.me>
Forwarded: not-needed
Last-Update: 2016-10-21
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -1,4 +0,0 @@
-browser:
-	node ./support/compile
-
-.PHONY: browser
\ No newline at end of file
